src/sass/layout/_header.scss

```diff
@@ -0,0 +1,157 @@
+@use '../abstracts';
+@use './functions' as *;
+
+.header {
+ 
+    $ideal-headerwrapper-padding-top: 255;
```

Comment:
you probably forgot about units

Reply:
This number is used in css function, that uses pixels to calculate padding-top